.github/workflows/release-helm-chart.yml

```
@@ -47,5 +44,7 @@ jobs:
           helm package ./helm-chart
 
       - name: Push Helm chart to GitHub Container Registry
+        permissions:
```

actionlint (1.7.4)

unexpected key "permissions" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)